packages/usa-file-input/src/index.js

```diff
@@ -313,24 +389,18 @@ const handleChange = (e, fileInputEl, instructions, dropTarget) => {
     if (fileNames[i]) {
       reader.readAsDataURL(fileNames[i]);
     }
+  }
 
-    // Adds heading above file previews, pluralizes if there are multiple
-    if (i === 0) {
-      dropTarget.insertBefore(filePreviewsHeading, instructions);
-      filePreviewsHeading.innerHTML = `Selected file <span class="usa-file-input__choose">Change file</span>`;
-    } else if (i >= 1) {
-      dropTarget.insertBefore(filePreviewsHeading, instructions);
-      filePreviewsHeading.innerHTML = Sanitizer.escapeHTML`${
-        i + 1
-      } files selected <span class="usa-file-input__choose">Change files</span>`;
-    }
+  if (fileNames.length === 0) {
+    // Reset input aria-label with default message
+    fileInputEl.setAttribute("aria-label", defaultAriaLabelText);
+  }
 
-    // Hides null state content and sets preview heading class
-    if (filePreviewsHeading) {
-      instructions.classList.add(HIDDEN_CLASS);
-      filePreviewsHeading.classList.add(PREVIEW_HEADING_CLASS);
-    }
+  if (fileNames.length > 0) {
```

Comment:
```suggestion
  if (fileNames.length === 0) {
    // Reset input aria-label with default message
    fileInputEl.setAttribute("aria-label", defaultAriaLabelText);
  } else {
```

Reply:
Nice cleanup. Updated in: [aaecb66](https://github.com/uswds/uswds/pull/5213/commits/aaecb668aff54e4c8c88ad60c28205890846eb4f)